ansible/README.md

```diff
@@ -3,3 +3,13 @@
 In a production environment, some parts of the wire-server infrastructure (such as e.g. cassandra databases) are best configured outside kubernetes. Additionally, kubernetes can be rapidly set up with a project called kubespray, via ansible.
 
 This directory hosts a range of ansible playbooks to install kubernetes and databases necessary for wire-server. For documentation on usage, please refer to the [Administrator's Guide](https://docs.wire.com), notably the production installation.
+
+
+## Bootrap environment created by `terraform/environment`
+
+1. Ensure `$ENV_DIR/operator-ssh.dec` exists and contains ssh key for the
+   environment.
+1. Ensure that `make apply` has been run for the environment. Please refer to
+   [terraform docs](../terraform/README.md) for details about how to run this.
+1. Ensure all required variables are set in `$ENV_DIR/inventory.yml`
```

Comment:
This is a difficult to solve problem, I think. Even if I write down all of them here, maintaining them will be a nightmare.
Also, ansible will only fail when running specific tasks. So, I don't really know how to do this. Do you have any ideas?

Reply:
Maybe we could create an example, which we run in CI, so it can keep track of what is required. Still if something is not required anymore and is there, nothing will tell us that. But it can be a start. Is it ok if we solve this problem separately?

Reply:
> Is it ok if we solve this problem separately?

Ok course, I already approved this PR. I agree it's a hard problem, let's try to find something in follow-up PRs. Involving CI sounds like a good idea.